File: src/main/java/org/example/kalkulationsprogramm/dto/Artikel/KategorieCreateDto.java
package org.example.kalkulationsprogramm.dto.Artikel;

import lombok.Getter;
import lombok.Setter;
import org.example.kalkulationsprogramm.domain.LieferantRolle;

import java.util.Set;

@Getter
@Setter
public class KategorieCreateDto {
    private String bezeichnung;
    private Integer parentId;
    private Set<LieferantRolle> typischeRollen;
}

File: src/main/kotlin/org/example/kalkulationsprogramm/dto/Artikel/KategorieCreateDto.kt
package org.example.kalkulationsprogramm.dto.Artikel

import org.example.kalkulationsprogramm.domain.LieferantRolle

data class KategorieCreateDto(
    var bezeichnung: String? = null,
    var parentId: Int? = null,
    var typischeRollen: Set<LieferantRolle>? = null,
)
